### DISCUSSION TITLE
1Password chrome extension not working

### DISCUSSION THREAD

**Discussion by project-guru** [dmarmor/epichrome#322] [General]
Just installed Epichrome for the first time. 

I tried installed the 1Password chrome extension but when I first opened the new app I created I got a message from 1Password saying that it cannot recognize the Browser.

Also, clicking on the extension icon doesn't work either and looks like this:

![2021-04-18_10-32-12](https://user-images.githubusercontent.com/18504294/115151174-5aac1200-a031-11eb-91c5-2eaa0729b2f8.png)

It should look like this:

![2021-04-18_10-33-35](https://user-images.githubusercontent.com/18504294/115151241-a959ac00-a031-11eb-87ca-6de892ebd780.png)

I researched this a bit an according to the 1Password site https://support.1password.com/code-signature/ it's something to do with code signature. 

Also per the same article, I tried moving my new app outside the Epichrome folder (so it's now in the Applications folder) but no luck.

Any way to make this work and would it be safe?

Thank you.

**Comment by dmarmor** [comment]
Is this a Brave app? If so, the desktop version of the 1Password extension will _never_ work. The way the built-in engine works, apps cannot pass 1Password's browser integrity check.

If you want to use the extension, you would need to create an app that uses the Chrome engine, and that app would need to live somewhere under the `/Applications` folder.

**Reply by project-guru** [reply]
Yes. It's using the Brave. 

On a related note, I just read that if I use the "Browser Only" version of 1Password called "1Password X" it will work. But I'm reluctant to do it this way because when signing into 1Password X for the first time, it's asking me for user name, password and Secret Key. Basically if somehow I was intercepted or hacked, they would have too much of my info including financial. 

Does anyone else feel this way?  How safe is this combination of a Pseudo app + 1Password X?

Thanks!

**Comment by dmarmor** [comment]
I use the 1PasswordX extension in a few of my own Brave apps. I suppose it's true an attacker could get at that info on first login, though it doesn't strike me as any more risky than using the extension with a full browser, since that's essentially what an Epichrome app is. And it's not really anything an attacker couldn't get with a keylogger the first time you logged into the desktop app, I suppose. But yeah, it's probably marginally less secure than using the desktop version (which I prefer anyway for a number of reasons, including that it allows me to use touch ID and that 1PasswordX's hotkeys don't work in app-style windows).
